tools/kicad/kicad_rs/src/eval/display.rs

```diff
@@ -0,0 +1,109 @@
+use evalexpr::Value;
+use std::fmt::{Display, Formatter, Result};
+
+const UNITS_LARGE: &[char] = &['k', 'M', 'G', 'T', 'E'];
+const UNITS_SMALL: &[char] = &['m', 'µ', 'n', 'p'];
```

Comment:
Could add the rest of the prefixes while we're at this. IMO at least femto (1e-15) is a somewhat commonly encountered prefix missing from the list.